### PR TITLE
Adding custom domain names

### PR DESCRIPTION
**Pull request by jbevan4**
Added custom domain names for db

### CODE REVIEW

File: modules/admin/route53.tf [hunk 1]
```diff
@@ -24,3 +24,11 @@ resource "aws_route53_record" "admin_alb_verification" {
   type   = tolist(aws_acm_certificate.admin_alb.domain_validation_options)[0].resource_record_type
 }
 
+resource "aws_route53_record" "admin_db" {
+  zone_id =  var.vpn_hosted_zone_id
+  ttl = 60
```
Comment (neilkidd) [inline modules/admin/route53.tf:29]: Is there a reason for a 60 seconds ttl? I thought 3600 was a typical value.

File: modules/dhcp/route53.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+resource "aws_route53_record" "dhcp_db" {
+  zone_id =  var.vpn_hosted_zone_id
+  ttl = 60
```
Comment (neilkidd) [inline modules/dhcp/route53.tf:3]: (Same question as above.) Is there a reason for a 60 seconds ttl? I thought 3600 was a typical value.

**Review by neilkidd — COMMENTED** [review]
Looks good - questions on the TTL values

File: modules/admin/route53.tf (hunk 1 above)
Comment (jbevan4) [reply modules/admin/route53.tf:29]: Another resource we had was 60 so I took inspiration from that. Happy to change this to 3600 if needed.

**Pull request merged**